fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/JobParameter.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * (C) Copyright IBM Corp. 2019, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.operation.bulkdata.model;
+
+public class JobParameter {
```

Comment:
the pattern that is used in the fhir-model is to have a static class. I think we should model that, not introduce a divergent pattern.

Reply:
just didn't feel too good to see the large piece of duplicated codes there ...

Reply:
Right, the fhir-model approach is what dictated the pattern. you can extend a class with the JobParameters subclass.